Initium-Odp/src/com/universeprojects/miniup/server/model/UpdateGridCell.java

```diff
@@ -0,0 +1,44 @@
+package com.universeprojects.miniup.server.model;
+
+public class UpdateGridCell {
```

Comment:
If you make the change I suggested for the GridCell class, you could just use that for updates.